Detected kernel version 3.10.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
wandb: Currently logged in as: jlbaker361 to https://api.wandb.ai. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.11
wandb: Run data is saved locally in /scratch/jlb638/wandb/wandb/run-20251005_225110-wgsfybx4
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fine-spaceship-170
wandb: ‚≠êÔ∏è View project at https://wandb.ai/jlbaker361/evaluation-creative
wandb: üöÄ View run at https://wandb.ai/jlbaker361/evaluation-creative/runs/wgsfybx4
Traceback (most recent call last):
  File "/cache/home/jlb638/Desktop/style-rl/compare_scores.py", line 55, in <module>
    main(args)
  File "/cache/home/jlb638/Desktop/style-rl/compare_scores.py", line 33, in main
    dataset=load_dataset(d,split="train").cast_column("image",Image).cast_column("augmented_image",Image)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/jlb638/conda_envs/deephands/lib/python3.11/site-packages/datasets/fingerprint.py", line 442, in wrapper
    out = func(dataset, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/jlb638/conda_envs/deephands/lib/python3.11/site-packages/datasets/arrow_dataset.py", line 2111, in cast_column
    dataset._data = dataset._data.cast(dataset.features.arrow_schema)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/jlb638/conda_envs/deephands/lib/python3.11/site-packages/datasets/features/features.py", line 1826, in arrow_schema
    return pa.schema(self.type).with_metadata({"huggingface": json.dumps(hf_metadata)})
                     ^^^^^^^^^
  File "/scratch/jlb638/conda_envs/deephands/lib/python3.11/site-packages/datasets/features/features.py", line 1815, in type
    return get_nested_type(self)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/jlb638/conda_envs/deephands/lib/python3.11/site-packages/datasets/features/features.py", line 1252, in get_nested_type
    return pa.struct(
           ^^^^^^^^^^
  File "pyarrow/types.pxi", line 5410, in pyarrow.lib.struct
  File "pyarrow/types.pxi", line 3894, in pyarrow.lib.field
  File "pyarrow/types.pxi", line 5922, in pyarrow.lib.ensure_type
TypeError: DataType expected, got <class 'datasets.features.image.Image'>
srun: error: gpu027: task 0: Exited with exit code 1
